src/panels/config/automation/ha-automation-sidebar.ts

```diff
@@ -32,99 +32,124 @@ export default class HaAutomationSidebar extends LitElement {
 
   @property({ type: Boolean }) public disabled = false;
 
-  @state() private _yamlMode = false;
+  @property({ type: Boolean }) public narrow = false;
 
-  @query(".sidebar-editor")
-  public editor?: HaAutomationTriggerEditor | HaAutomationConditionEditor;
+  @state() private _yamlMode = false;
 
-  protected render() {
-    if (!this.config) {
-      return nothing;
-    }
+  @query("ha-bottom-sheet") private _bottomSheetElement?: HaBottomSheet;
 
+  private _renderContent() {
     // get config type
     const type = this._getType();
 
     if (type === "trigger") {
       return html`
         <ha-automation-sidebar-trigger
+          class="sidebar-content"
```

Comment:
Why not wrap the content in a div that handles this styling when using the sidebar, instead of adding the class to every element?

Reply:
Because we need the height 100% on the sidebar element itself, a wrapper breaks it